tools/depends/configure.ac

```diff
@@ -22,6 +22,12 @@ AC_DEFUN([MC_CHECK_NOT_CPU],
   esac
 ])
 
+# default android sdk / ndk versions
+if expr "$host" : '.*-android.*' ; then
+  use_sdk=android-26
+  use_ndk_api=21
+fi
```

Comment:
what's wrong with the previous place of this?

Reply:
You have to edit 3 places, and risk is there that you forget one while editing.